elements/lisk-p2p/src/p2p.ts

```diff
@@ -125,6 +125,7 @@ const BASE_10_RADIX = 10;
 const DEFAULT_MAX_OUTBOUND_CONNECTIONS = 20;
 const DEFAULT_MAX_INBOUND_CONNECTIONS = 100;
 const DEFAULT_OUTBOUND_SHUFFLE_INTERVAL = 300000;
+const DEFAULT_EVICTION_PROTECTION = 0.58;
```

Comment:
Does this mean that we protect 58% of peers in our list?
If so, it seems like a lot. Can you double check with Jan?

Reply:
The percentage is derived from the following [bitcoin strategy of eviction](https://github.com/bitcoin/bitcoin/blob/17d644901bbd770578619545bbd8bc930fe6f2d9/src/net.cpp#L1000). Bitcoin allows maximum 117 incoming connections. Out of those, 4 are protected by `netgroup`, 8 by `latency`, 8 by `useful work`, making 20 peers. Half of the remaining peers (97 / 2 = 48.5 peers) are protected for connected time. (20 + 48.5) / 117 = ~0.58